src/pages/strategies/create/overlapping.test.tsx

```diff
@@ -0,0 +1,70 @@
+import { test, expect, afterEach, vitest } from 'vitest';
+import { renderWithRouter, screen } from 'libs/testing-library';
+import { debugTokens } from '../../../../e2e/utils/types';
+import { CreateOverlappingStrategyPage } from './overlapping';
+import userEvent from '@testing-library/user-event';
+
+vitest.mock('hooks/useMarketPrice', () => {
+  return {
+    useMarketPrice: () => {
+      return { marketPrice: 2800, isPending: false };
+    },
+  };
+});
+
+const basePath = '/strategies/create/overlapping';
+afterEach(async () => {
+  window.history.replaceState(null, 'root', '/');
+});
+
+test('run overlapping form with deeplink and user market price', async () => {
+  // Initialize user and set params
+  const user = userEvent.setup();
+  const search = {
+    base: debugTokens.ETH,
+    quote: debugTokens.USDC,
+    marketPrice: '2500',
+    min: '2000',
+    max: '3000',
+    spread: '0.01',
+    anchor: 'buy',
+    budget: '100',
+  };
+
+  // Render page
+  const { router } = await renderWithRouter({
+    component: () => <CreateOverlappingStrategyPage />,
+    basePath,
+    search,
+  });
+
+  // Check search params
+  expect(router.state.location.pathname).toBe(basePath);
+  expect(router.state.location.search).toStrictEqual(search);
+
+  // Close price chart
+  await user.click(screen.getByText('Close Chart'));
```

Comment:
As discussed in the channel, I don't like using text or role query. I think `data-testid` is more consistent. Also it ensure separation of concern : 
- query with testid make sure you have to element
- check role or text with except to make it explicit that you want to verify this information